added content guidelines to org repo, fixed links and formatting in d…

stacymcauliffe opened the pull request:
I moved content guidelines to the org repo

Also updated links to follow new sitemap from Joachim

Finally, updated formatting for existing design guidelines to match template

There is additional work that needs to be done to round out the button and nav system content that is not contained by this PR.

mcarrano commented on packages/patternfly-4/src/content/design-guidelines/content/writing.md:
```diff
@@ -0,0 +1,123 @@
+
+During this stage, it's also useful to pull all content out of the wireframes and catalog it using a spreadsheet or doc. This step is especially important when you are working with a larger team and content is reviewed and collaborated on by multiple people. 
+
+![Copy doc example](/img/copydoc.png)
```
I get a 404 error on this link. Does this example exist?

stacymcauliffe commented:
@mcarrano  -- Thanks! I fixed the broken link. 

The content is all the same, but I updated almost every file to match the template and updated a lot of broken links across all of the design guidelines.
